src/Modules/UsersModule/components/UsersList/UsersList.tsx

```diff
@@ -1,5 +1,378 @@
+                </span>
+              </p>
+            </div>
+          </Modal.Body>
+          <ModalFooter className="py-3">
+            <button
+              onClick={handleCloseShowUser}
+              className={`btn py-1 px-3 fs-6 fw-medium btn btn-outline-success `}
+            >
+              Back
+            </button>
+          </ModalFooter>
+        </Modal>
+
+        {/*Filtaration*/}
+        <div className="container">
+          <div className="input-container w-25 ">
+            <input
+              placeholder="Search By Name "
+              className={`input-field input-theme`}
+              type="text"
+              onChange={handleSearchChange}
+            />
+            <label htmlFor="input-field" className={`input-label `}>
+              Search
+            </label>
+            <span className="input-highlight"></span>
+          </div>
+        </div>
+
+        {/* table */}
+        <div
+          className={`project-body head-bg mt-5 container rounded-4 shadow px-4 py-5`}
+        >
+          <ul className={`${style.responsiveTableProjects} text-white`}>
+            <li className={`${style.tableHeader}`}>
+              <div className={`${style.col} ${style.col1}`}>ID</div>
+              <div className={`${style.col} ${style.col2}`}>UserName</div>
+              <div className={`${style.col} ${style.col3}`}>Image</div>
+              <div className={`${style.col} ${style.col4}`}>Creation Date</div>
+              <div className={`${style.col} ${style.col5}`}>Actions</div>
+              <div className={`${style.col} ${style.col6}`}>Status</div>
+            </li>
+          </ul>
+          {isLoading ? (
+            <div className="container pt-5 mt-5 d-flex justify-content-center ">
+              {loader()}
+            </div>
+          ) : (
+            <ul className={`${style.responsiveTableProjects}`}>
+              {usersList.length > 0 ? (
+                usersList.map((user: UsersInterface) => (
+                  <li
+                    key={user.id}
+                    className={`${style.tableRow} bg-theme text-theme`}
+                  >
+                    <div
+                      className={`${style.col} ${style.col1}`}
+                      data-label="ID"
+                    >
+                      {user.id}
+                    </div>
+                    <div
+                      className={`${style.col} ${style.col2}`}
+                      data-label="Title :"
+                    >
+                      {user.userName}
+                    </div>
+                    <div
+                      className={`${style.col} ${style.col3}`}
+                      data-label="Images :"
+                    >
+                      {user.imagePath ? (
+                        <img
+                          className={`${style.noImg} rounded-5`}
+                          src={`https://upskilling-egypt.com:3003/${user.imagePath}`}
```

Comment:
We need to make an imagePath URL. If the url changes, we will need to update it manually in all locations.